bundles/org.openhab.binding.zway/src/main/resources/OH-INF/i18n/zway.properties

```diff
@@ -1,15 +1,15 @@
-# binding
-binding.zway.name = Z-Way Binding
-binding.zway.description = Z-Way is a home automation software to configure and control a Z-Wave network. The ZAutomation interface provides all Z-Wave devices and handles incoming commands. The Z-Way Binding uses this HTTP interface to load all device and make them available during the discovery process.<br> Currently only a continuous polling is available!
+# add-on
+addon.zway.name = Z-Way Binding
+addon.zway.description = Z-Way is a home automation software to configure and control a Z-Wave network. The ZAutomation interface provides all Z-Wave devices and handles incoming commands. The Z-Way Binding uses this HTTP interface to load all device and make them available during the discovery process.<br> Currently only a continuous polling is available!
 
 # thing types
 thing-type.zway.zwayServer.label = Z-Way Server
 thing-type.zway.zwayServer.description = The Z-Way server represents a bridge with general settings and communication tasks.
 
 thing-type.config.zway.zwayServer.zwayServer.label = Z-Way server
 thing-type.config.zway.zwayServer.zwayServer.description = The configuration of the Z-Way server. Except for the username and password all the information detected during the discovery.
-thing-type.config.zway.zwayServer.binding.label = Options
-thing-type.config.zway.zwayServer.binding.description = These settings affect functions of the binding.
+thing-type.config.zway.zwayServer.addon.label = Options
```

Comment:
Possibly same issue as in my comment on Velux ??

Reply:
See above.